src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -542,7 +564,7 @@ private static JsonSerializerOptions GetFullyPopulatedOptionsInstance()
         {
             var options = new JsonSerializerOptions();
 
-            foreach (PropertyInfo property in typeof(JsonSerializerOptions).GetProperties())
+            foreach (PropertyInfo property in typeof(JsonSerializerOptions).GetProperties(BindingFlags.Public | BindingFlags.Instance))
```

Comment:
Introduce variable for properties because the line is long?
e.g. `var properties = typeof(JsonSerializerOptions).GetProperties(BindingFlags.Public | BindingFlags.Instance)`